src/status_im/contexts/onboarding/enable_notifications/view.cljs

```diff
@@ -17,51 +21,111 @@
     :description                     (i18n/label :t/enable-notifications-sub-title)
     :description-accessibility-label :notifications-sub-title}])
 
-(defn enable-notification-buttons
-  [{:keys [insets]}]
-  (let [profile-color   (rf/sub [:onboarding/customization-color])
-        ask-permission  (fn []
-                          (rf/dispatch
-                           [:request-notifications
-                            {:on-allowed (fn []
-                                           (js/setTimeout
-                                            #(rf/dispatch [:onboarding/finish-onboarding true])
-                                            300))
-                             :on-denied  (fn []
-                                           (js/setTimeout
-                                            #(rf/dispatch [:onboarding/finish-onboarding false])
-                                            300))}]))
-        skip-permission #(rf/dispatch [:onboarding/finish-onboarding false])]
-    [rn/view {:style (style/buttons insets)}
-     [quo/button
-      {:on-press            ask-permission
-       :type                :primary
-       :icon-left           :i/notifications
-       :accessibility-label :enable-notifications-button
-       :customization-color profile-color}
-      (i18n/label :t/intro-wizard-title6)]
-     [quo/button
-      {:on-press            skip-permission
-       :accessibility-label :enable-notifications-later-button
-       :type                :grey
-       :background          :blur
-       :container-style     {:margin-top 12}}
-      (i18n/label :t/maybe-later)]]))
+(defn on-notifications-setup-start
+  [params]
+  (rf/dispatch [:onboarding/notifications-setup-start params]))
 
-(defn enable-notifications-simple
+(defn notifications-info-view
+  [{:keys [blur?]}]
+  [quo/documentation-drawers
+   {:title        (i18n/label :t/enable-notifications)
+    :show-button? true
+    :shell?       blur?
+    :button-label (i18n/label :t/read-more)
+    :button-icon  :i/info}
+   [quo/text (i18n/label :t/enable-notifications-info-description)]])
```

Comment:
This drawer view still needs the correct verbiage from the docs team (I think ?)
Reference: https://www.figma.com/design/JlpPhJp0SMnEyBQy4nOZHo/Settings----Mobile?node-id=9395-120255&t=IZIHOX0Oys0QGyTf-4